app/assess/routes.py

```diff
@@ -184,12 +113,89 @@ def display_sub_criteria(
 
     return render_template(
         "sub_criteria.html",
-        on_summary=False,
         answers_meta=answers_meta,
         **common_template_config,
     )
 
 
+@assess_bp.route(
+    "/application_id/<application_id>/sub_criteria_id/<sub_criteria_id>/score",
+    methods=["POST", "GET"],
+)
+@login_required(roles_required=["LEAD_ASSESSOR", "ASSESSOR"])
+def score(
+    application_id,
+    sub_criteria_id,
+):
+    sub_criteria: SubCriteria = get_sub_criteria(
+        application_id, sub_criteria_id
+    )
+
+    if not sub_criteria.is_scored:
+        abort(404)
+    theme_id = sub_criteria.themes[0].id
+    fund = get_fund(Config.COF_FUND_ID)
+    flag = get_latest_flag(application_id)
+    comments = get_comments(
```

Comment:
is the way we retrieve comments here correct for the scoring route?

Reply:
To return comments for a score it looks like the assessment store is set up to retrieve 'theme_id = "score' https://github.com/communitiesuk/funding-service-design-assessment-store/blob/main/db/queries/comments/queries.py#:~:text=if%20theme_id%20%3D%3D%20%22score%22%3A

Reply:
@Adam-W1 Good spot 👀 
Sad we have included this concept in the API too 😢 
Have fixed and added TODO in 023370a.